tox.ini

```diff
@@ -41,3 +41,5 @@ commands =
 commands = flake8 --extend-ignore F401 {posargs:jedi}
 deps =
 extras = qa
+[pycodestyle]
+max-line-length=100
```

Comment:
OK. But can we have the `pycodestyle` section alongside with the `flake8` section in this case? Or this change is for different PR?

Reply:
Oh now I realized that it's not the same "directive". If it's possible to use in `setup.cfg` I'm fine with that. If it's not we can also keep it in tox.ini. But since I want to get rid of tox eventually I'd rather not have it in that file.